mscore/importgtp.cpp

```diff
@@ -151,10 +151,9 @@ void GuitarPro::read(void* p, qint64 len)
       if (len == 0)
             return;
       qint64 rv = f->read((char*)p, len);
-#ifndef NDEBUG
-      Q_UNUSED(rv); // avoid warning about unused variable in RELEASE mode
-      Q_ASSERT(rv == len);
-#endif
+      if (rv != len) {
+            Q_ASSERT(0);
```

Comment:
wouldn't it make sense to  still use `Q_ASSERT(rv == len);` here, for a decent assert message?

Reply:
Why not :) Also I will add comment to clarify such changes.